mono.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: mono
+  version: 6.14.0
+  epoch: 0
+  description: The mono framework.
+  copyright:
+    - license: Apache-2.0
+    - license: MIT
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  environment:
+    CFLAGS: "-Wno-int-conversion -DMONO_ARCH_NOMAP32BIT"
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bash
+      - build-base
+      - busybox
+      - cmake
+      - gdb
+      - gettext
+      - krb5
+      - libtool
+      - python3
+      - wget
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.winehq.org/mono/mono.git
+      tag: mono-${{package.version}}
+      expected-commit: 3d1eb55c535487d608ef4b57aca330803a6e002d
+      destination: /home/build/src
+
+  - working-directory: /home/build/src
```

Comment:
I think we can just use the default /home/build working directory, same for the `git-checkout`'s `destination`. WDYT?